src/Setup/WixBA/WixBA.csproj

```diff
@@ -1,19 +1,50 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information. -->
-
-
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="12.0">
   <PropertyGroup>
     <ProjectGuid>{7C27518B-84AD-4679-8EF4-29DF552CF1AC}</ProjectGuid>
     <AssemblyName>WixBA</AssemblyName>
     <OutputType>Library</OutputType>
     <RootNamespace>WixToolset.UX</RootNamespace>
+    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
```

Comment:
I really don't know.  The only reason that the PR had a diff here is because I experimented with compiling to NetFx45 and then after the online meeting 109 I reverted back to NetFx40 as requested.  I will look at  tools\WixBuild.csproj.props and remove this block if that seems wise.